backend/app/assets/stylesheets/spree/backend/shared/_layout.scss

```diff
@@ -1,16 +1,12 @@
-// Basics
-//---------------------------------------------------
-* {
-  -webkit-box-sizing: border-box;
-     -moz-box-sizing: border-box;
-          box-sizing: border-box;
-}
-
-// Page Layout
-//---------------------------------------------------
-
 html {
   height: 100%;
+  box-sizing: border-box;
+}
+
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
```

Comment:
All of this is provided by bootstrap's `_reboot.scss`. It can probably all be removed.